.travis.yml

```diff
@@ -1,52 +1,61 @@
 language: python
-dist: trusty
-sudo: required
-python: 3.6
-before_install:
-  - sudo add-apt-repository -y ppa:fkrull/deadsnakes
-  - sudo apt-get -qq update
-  - sudo apt-get -y install python2.6 python3.2
-install: pip install "virtualenv<14.0.0"
-script: python setup.py test
+sudo: false
+python:
+  - 2.6
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - 3.5
+  - 3.6
 env:
-  # To generate (update) the env list run
-  # $ tox -l | sort | xargs -I ITEM echo "  - TOXENV="ITEM
-  # then manually sort the output so that the TOXENV values are in
-  # order by Python version and Django version
-  - TOXENV=flake8-py26
-  - TOXENV=flake8-py27
-  - TOXENV=flake8-py32
-  - TOXENV=flake8-py33
-  - TOXENV=flake8-py34
-  - TOXENV=flake8-py35
-  - TOXENV=pylint-py27-django111
-  - TOXENV=pylint-py36-django111
-  - TOXENV=py26-django14
-  - TOXENV=py26-django15
-  - TOXENV=py26-django16
-  - TOXENV=py27-django14
-  - TOXENV=py27-django15
-  - TOXENV=py27-django16
-  - TOXENV=py27-django17
-  - TOXENV=py27-django18
-  - TOXENV=py27-django19
-  - TOXENV=py27-django110
-  - TOXENV=py27-django111
-  - TOXENV=py32-django15
-  - TOXENV=py32-django16
-  - TOXENV=py32-django17
-  - TOXENV=py32-django18
-  - TOXENV=py33-django15
-  - TOXENV=py33-django16
-  - TOXENV=py33-django17
-  - TOXENV=py33-django18
-  - TOXENV=py34-django17
-  - TOXENV=py34-django18
-  - TOXENV=py34-django19
-  - TOXENV=py34-django110
-  - TOXENV=py34-django111
-  - TOXENV=py35-django18
-  - TOXENV=py35-django19
-  - TOXENV=py35-django110
-  - TOXENV=py35-django111
-  - TOXENV=py36-django111
+  - DJANGO=flake8
```

Comment:
Remove this line, please, and let's use an `include:` in the matrix below. Let's settle on, it's sufficient to run flake8 and Pylint over the code base once (either just with Python 3.6, optionally with Python 2.7 in addition). -- See comment below!